app/cdap/components/Replicator/Create/index.tsx

```diff
@@ -471,6 +507,24 @@ class CreateView extends React.PureComponent<ICreateProps, ICreateContext> {
 
     return true;
   };
+  /**
+   * If the tink plugin is in the list of artifacts,
+   * enable the tink option for transformations
+   */
+  private isTinkEnabled = () => {
```

Comment:
Nit: `isFoo` implies that the function will return a boolean with no side effects. Can you consider renaming this to `syncTinkEnabled`?

(Actually, that rhyme is terrible. But you get the idea.)